ibm/mas_devops/playbooks/ocp_rosa_provision.yml

```diff
@@ -2,7 +2,7 @@
 - hosts: localhost
   vars:
     cluster_type: rosa
-    ocp_version: "{{ lookup('env', 'OCP_VERSION') | default('4.10.18', True) }}"
+    ocp_version: "{{ lookup('env', 'OCP_VERSION') | default('4.14.5', True) }}"
```

Comment:
could you please double chekc the version of 4.14.x rosa supports, is it 4.14.12?

Reply:
There is no specific versions for ROSA. In their docs they specify guidelines as per https://access.redhat.com/documentation/en-us/red_hat_openshift_service_on_aws/4/html/introduction_to_rosa/policies-and-service-definition#rosa-life-cycle

Reply:
Using the latest with as of today is 4.14.16

Reply:
we are using 4.14.12 in fvt and airgap mirror.. lets stick with that.